README.rst

```diff
@@ -74,14 +89,70 @@ Each editor defined in ``WAGTAILADMIN_RICH_TEXT_EDITORS`` is a dictionary with 2
 -  ``WIDGET`` is a mandatory string set to the widget to use
    -  should always be set to ``wagtaildraftail.widgets.DraftailTextArea`` (or a subclass of it) to work with Draft.js content
 -  ``OPTIONS`` is a dictionary which follows the format of `Draftail configuration options <https://github.com/springload/draftail#usage>`_.
-   -  Draftail options which are JavaScript values are hydrated at runtime in ``client/wagtaildraftail.js``
+   -  Draftail options which are JavaScript values are hydrated at runtime in ``client/wagtaildraftail.js``. Alternatively, on Wagtail 1.12 and above, a ``features`` list can be passed within the ``OPTIONS`` dict in place of a full Draftail configuration.
 
 **WARNING:** The ``type`` key for ``blockTypes``, ``inlineStyles`` and ``entityTypes`` shouldn’t be changed. It is what defines how content is rendered, and is saved as a JSON blob in the database which would make migrations really painful.
 
 **WARNING:** All the blocks/styles/entities defined in the editor config should have been configured to render properly in the `exporter config <#exporter-configuration>`_.
 
 Here is a sample configuration file. This should live in your Django settings.
 
+For Wagtail 1.12.x and above:
+
+.. code:: python
+
+    from draftjs_exporter.constants import BLOCK_TYPES, ENTITY_TYPES
+    from draftjs_exporter.defaults import BLOCK_MAP
+
+    WAGTAILADMIN_RICH_TEXT_EDITORS = {
```

Comment:
Agreed. Since this change makes the Draftail editor fully functional with no extra configuration, I looked at whether it would be possible to leave out the `WAGTAILADMIN_RICH_TEXT_EDITORS` setting entirely... but for that to work, we have to inject a `'default_draftail'` entry into the `wagtail.wagtailadmin.rich_text.DEFAULT_RICH_TEXT_EDITORS` dict before calling `get_rich_text_editor_widget`.

Anyway, I think this problem goes away when we make Draftail a drop-in replacement for hallo.js - at that point, running two editors side by side becomes the exceptional case rather than the normal one (and so it makes sense to show them both explicitly in the config).

Incidentally, I'm 99% sure you don't need the `custom` entry on a real site - I don't think that's even a working editor implementation. It only exists so that we can test that importing alternative widgets actually works, while there's only one defined in the Wagtail codebase :-)